upload_form.py

```diff
@@ -345,15 +345,15 @@ def set_columns(filename, fileformat=None):
 
     append_table(merged_table, table)
     Table.write(merged_table, merged_table_name, format='ascii.ipac')
-
+    
     username = column_data.get('Username')['Name']
     branch,timestamp = commit_change_to_database(username)
     time.sleep(2)
-    # Adding raw file to uploads
-    #dummy = commit_change_to_database(username, tablename=filename, workingdir='uploads/',database='upl')
-    #time.sleep(2)
     pull_request(branch, username, timestamp)
-    #pull_request(branch, username, timestamp, database='uploads')
+    # Adding raw file to uploads
+    dummy = commit_change_to_database(username, tablename=filename, workingdir='uploads/',database='upl',branch=branch)
```

Comment:
Yes, the problem is that the `.git` suffix is missing in `.gitmodules`.  I think if we just add it there, it will fix this problem.